akmodsposttrans@.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Building and installing kmods for kernel %i
+
+[Service]
+Type=simple
+RemainAfterExit=yes
+ExecStart=/usr/bin/systemd-inhibit --who=akmods --why="Building kernel modules" /usr/sbin/akmods --from-kernel-posttrans --kernels %i
```

Comment:
Can you verify that we need systemd-inihibit if we switch from nohup script to systemd service ?
What I expect is that since the the systemd service will be running it will prevent the shutdown process to kill the service...